<launch>
    <node pkg="chess_ur_pkg" type="ur_get_point" name="ur_get_point" output="screen" >
        <param name="srv_name" value="ur_get_point"  />
        <param name="group_name" value="manipulator" />
    </node>
</launch>